src/trelloHandler.js

```diff
@@ -88,13 +88,20 @@ window.trelloHandler = new Vue({
       const vm = this;
       this.loading = true;
 
+      if (!this.selectedList && !this.selectedLabel){
+        this.loading = false;
+        return;
```

Comment:
After a deeper review, my spider sense kind of felt a bug somewhere because of this. If there is no bug, it's thanks to JS shadiness with its sloppy comparisons (null != undefined).

This return states the wrong intention: it should at least return a dummy promise. And since now you can both select List and Label, you might do something on their state.

Moreover, their states are supposed to be fetch from and persisted in the local storage. What I've seen, you have either a list or a label so the reload may fail or incompletely fill the app state.

Could we see each other to check my assumptions and find the best solution together? :)